.github/workflows/non-cot-pr-build-and-e2e-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Run tests against PR in an environment with COT enabled
+name: Run tests against PR in an environment with COT disabled
```

Comment:
I would take advantage of this change to rename the COT occurrences to HPOS:
```
name: Run tests against PR in an environment with HPOS disabled
```
Or even:
```
name: Run tests against PR in an environment without HPOS
```

Reply:
Marking this as resolved now. I can already see the name as `Run tests against PR in an environment with HPOS disabled` which is much more appropriate than `...without HPOS`.